examples/express-ex/package.json

```
@@ -14,7 +14,7 @@
   "author": "fabiojose@gmail.com",
   "license": "Apache-2.0",
   "dependencies": {
-    "cloudevents-sdk": "github:cloudevents/sdk-javascript#v1",
+    "cloudevents-sdk": "1.0.0",
```

Maybe it would be good to use a [patch or minor release filter](https://docs.npmjs.com/about-semantic-versioning#using-semantic-versioning-to-specify-update-types-your-package-can-accept) here?

```suggestion
    "cloudevents-sdk": "~1.0.0",
```

---

Probably yes :+1:

---

Commit updated